CMakeLists.txt

```diff
@@ -42,6 +42,9 @@ elseif(NOT WIN32)
         option(HIDAPI_WITH_HIDRAW "Build HIDRAW-based implementation of HIDAPI" ON)
         option(HIDAPI_WITH_LIBUSB "Build LIBUSB-based implementation of HIDAPI" ON)
     endif()
+    if(CMAKE_SYSTEM_NAME MATCHES "NetBSD")
```

Comment:
Just to double-check: is `uhid` backend specific to NetBSD or is it available on other BSD systems?

Reply:
Technically it is available to other BSD systems since it has existed since sometime in 1990. But they have diverged heavily over the years so they are all at varying levels of compatibility not to mention differences in internal kernel behavior. The compatible subset I found is extremely limited and would not make for a very useful generic approach. I thought it best to combine `uhid` with platform specific support code to help workaround its limitations. NetBSD probably has the most advanced `uhid` implementation in terms of overall features.

Reply:
Thanks, this is helpfull.
I agree, better keep it for NetBSD only.